shadows/framework/src/main/java/org/robolectric/shadows/ShadowUserManager.java

```diff
@@ -77,40 +76,38 @@ public class ShadowUserManager {
   public static final int FLAG_SYSTEM = UserInfo.FLAG_SYSTEM;
   public static final int FLAG_MAIN = UserInfo.FLAG_MAIN;
 
-  private static int maxSupportedUsers = DEFAULT_MAX_SUPPORTED_USERS;
-  private static boolean isMultiUserSupported = false;
-  private static boolean isHeadlessSystemUserMode = false;
-
   private final Object lock = new Object();
 
   @RealObject private UserManager realObject;
-  private UserManagerState userManagerState;
-  private Boolean managedProfile;
-  private Boolean cloneProfile;
-  private boolean userUnlocked = true;
-  private boolean isSystemUser = true;
-  private volatile boolean isForegroundUser = true;
+  private static UserManagerState userManagerState;
 
   /**
    * Holds whether or not a managed profile can be unlocked. If a profile is not in this map, it is
    * assume it can be unlocked.
    */
```

Comment:
It looks like this comment it attached to the wrong field. Even before this change, I'm not sure it was meant to be for `seedAccountName`.